fog-google.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # As of 0.1.1
-  spec.required_ruby_version = "~> 2.0"
+  spec.required_ruby_version = ">= 2.0"
```

Comment:
I noticed a number of integration tests failing with Ruby 3.0. I think this is blocked by https://github.com/googleapis/google-api-ruby-client/issues/890?

Reply:
I just looked at googleapis/google-api-ruby-client#890 and it's working as intended. The original poster's usage was incorrect. (They were trying to pass hashes into constructors that took keyword arguments, which worked on Ruby 2.6, gave warnings on Ruby 2.7, and failed on Ruby 3.0.)